client/components/thank-you-v2/upsell/style.scss

```
@@ -2,150 +2,79 @@
 @import "@wordpress/base-styles/mixins";
 
 .thank-you__upsell {
-	display: flex;
-	width: 100%;
-	max-width: 650px;
+	border-radius: 2px;
+	border: 1px solid var(--color-border-subtle);
+	box-sizing: border-box;
+	display: grid;
+	grid-template-columns: 100px 1fr;
 	margin: 0 auto 80px;
-	flex-direction: column;
-	justify-content: center;
-	align-items: center;
-	gap: 12px !important;
-	text-align: left;
+	max-width: 650px;
+	overflow: hidden;
+	width: 100%;
 
-	.thank-you__upsell-title {
-		margin: 0 24px;
-		color: var(--gray-gray-100, #101517);
-		font-size: $font-body;
-		font-family: "SF Pro Text", $sans;
-		font-style: normal;
-		font-weight: 500;
-		line-height: 24px; /* 150% */
-		letter-spacing: -0.32px;
-		align-self: stretch;
-		@media (max-width: $break-medium ) {
-			margin: 0;
-		}
+	@media (max-width: $break-small ) {
+		grid-template-columns: 100%;
 	}
+}
 
-	.thank-you__upsell-content {
-		display: flex;
-		width: 100%;
-		padding-right: 0;
-		align-items: center;
-		gap: 20px;
-		border-radius: 2px;
-		border: 1px solid var(--gray-gray-5, #dcdcde);
+.thank-you__upsell-image {
+	display: flex;
 
-		@media (max-width: $break-mobile ) {
-			display: flex;
-			flex-direction: column;
-			justify-content: center;
-			align-items: start;
-			align-self: stretch;
-		}
+	@media (max-width: $break-small ) {
+		max-height: 80px;
 	}
 
-	.thank-you__upsell-content-details {
-		display: flex;
-		align-items: center;
-		gap: 16px;
-		flex: 1 0 0;
-		@media (max-width: $break-mobile ) {
-			display: flex;
-			flex-direction: column;
-			justify-content: center;
-			align-items: start;
-			gap: 12px;
-			align-self: stretch;
-		}
+	> img {
```

ATM it's unnecessary, and in general I don't see a reason to emphasize the first level of child
```suggestion
	img {
```

---

Making the change. Noting that I think this is a nit that comes down to personal preference more than having any actual impact, though.